docker/redshift/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM demisto/python3-ubi:3.10.8.90164
+ADD https://s3.amazonaws.com/redshift-downloads/drivers/odbc/2.1.0.0/AmazonRedshiftODBC-64-bit-2.1.0.0.x86_64.rpm redshift.rpm
+COPY odbcinst.ini /etc/odbcinst.ini
+COPY amazon.redshiftodbc.ini /etc/amazon.redshiftodbc.ini
+COPY requirements.txt .
+RUN yum -y install unixODBC && yum -y --nogpgcheck localinstall redshift.rpm
```

```suggestion
RUN dnf -y install unixODBC && yum -y --nogpgcheck localinstall redshift.rpm
```

use `dnf` it is better than yum, its more updated and has better performance